app/javascript/mastodon/locales/en.json

```diff
@@ -239,6 +239,7 @@
   "status.show_more": "Show more",
   "status.unmute_conversation": "Unmute conversation",
   "status.unpin": "Unpin from profile",
+  "status.pinned": "Pinned toot",
```

Comment:
Strings should be sorted alphabetically, run `yarn manage:translations`, you can also add Polish string for me – “Przypięty wpis”.

Reply:
added :D